openslides/agenda/projector.py

```diff
@@ -10,11 +10,41 @@
 
 class ItemListSlide(ProjectorElement):
     """
-    Slide definitions for the agenda.
+    Slide definitions for Item model.
+
+    This is only for list slides.
+
+    Omit 'id' to get the whole agenda tree. Set 'id' to None to get a list
+    slide of all root items. Set 'id' to an integer to get a list slide of
+    the children of the metioned item.
     """
-    name = 'agenda/agenda'
+    name = 'agenda/item-list'
+
+    def get_context(self):
+        pk = self.config_entry.get('id', 'tree')
+        if pk == 'tree':
+            # Agenda tree.
+            context = {'tree': True}
+        elif pk is None:
+            # Root list slide.
+            context = {'tree': False}
+        else:
+            # Children slide.
+            if not Item.objects.filter(pk=pk).exists():
+                raise ProjectorException(_('Item does not exist.'))
+            context = {'id': pk}
+        return context
 
     def get_requirements(self, config_entry):
+        pk = config_entry.get('id', 'tree')
+        if pk == 'tree':
+            # Agenda tree.
+            yield ProjectorRequirement(
+                view_class=ItemViewSet,
+                view_action='tree')
```

Comment:
The tree-rest-request is also needed when only the items with no parents are shown on a slide (pk=None in your implementation). The tree-view is the only way to get the information which items have no parents. (We excluded this information from the item-detail-view.)